libs/block-content/website/src/lib/teaser/teaser.tsx

```diff
@@ -162,20 +138,76 @@ export const selectTeaserAuthors = (teaser: TeaserType) => {
   }
 }
 
-const teaserImageStyles = css`
+export const TeaserWrapper = styled('article')<FlexAlignment>`
+  display: grid;
+
+  ${({w}) =>
+    w > 6 &&
+    css`
+      grid-column: 1 / -1;
+    `}
+
+  ${({theme, h, w, x, y}) => css`
+    ${theme.breakpoints.up('md')} {
+      grid-column-start: ${x + 1};
+      grid-column-end: ${x + 1 + w};
+      grid-row-start: ${y + 1};
+      grid-row-end: ${y + 1 + h};
+    }
+  `}
+`
+
+export const TeaserInnerWrapper = styled('div')`
+  display: grid;
+  gap: ${({theme}) => theme.spacing(1)};
+  grid-template-rows: auto;
+  grid-auto-rows: max-content;
+`
+
+export const ImageWrapperStyled = styled('div')`
+  grid-column: 1/13;
+  width: 100%;
   height: 100%;
+  overflow: hidden;
+`
+
+const teaserImageStyles = css`
   width: 100%;
   object-fit: cover;
+  grid-column: 1/13;
+  transition: transform 0.3s ease-in-out;
 `
 
-const teaserLinkStyles = css`
+const getEnhancedImageStyles = (theme: Theme, isHovered: boolean) => css`
```

Comment:
Probably want to do it like you did before in the navigation

Reply:
```
const useNavbarLinkStyles = () => {
  const theme = useTheme()

  return useMemo(
    () => css`
      font-size: 1rem;
      margin: 0 ${theme.spacing(1)} 0 ${theme.spacing(2)};
      text-decoration: none;
      color: ${theme.palette.common.black};

      ${theme.breakpoints.up('md')} {
        font-size: 1.3rem;
      }
    `,
    [theme]
  )
}
```

Like you done in the navbar. This way it doesn't rerender unnecessarily which is better than my example implementation (basically like this one).